zerver/models.py

```diff
@@ -768,6 +768,17 @@ def to_dict_cache_key(message, apply_markdown):
     # type: (Message, bool) -> text_type
     return to_dict_cache_key_id(message.id, apply_markdown)
 
+class Topic(ModelReprMixin, models.Model):
```

Comment:
Ok, I might do the comment as a separate commit, just because it's gonna churn a little bit.  (I could add a comment to this commit that says the table isn't even populated with data, just to have the subsequent commit say that it's populated with data but not linked back to message, etc.)